src/cnv/ugbio_cnv/plot_cnv_results.py

```diff
@@ -340,24 +346,38 @@ def run(argv):  # noqa: C901, PLR0912, PLR0915 # TODO: refactor
     ##### plot CNV calls #####
     ##########################
     # load UG calls
+    def extract_copy_number(info_string):
+        spstr = info_string.split(";")
+        spstr = [x for x in spstr if x.startswith("CopyNumber")][0]
+        return float(spstr.replace("CopyNumber=", ""))
+
     if args.duplication_cnv_calls:
         if os.path.getsize(args.duplication_cnv_calls) > 0:
             df_dup = pd.read_csv(args.duplication_cnv_calls, sep="\t", header=None)
-            df_dup.columns = ["chr", "start", "end", "copy-number"]
+            if args.vcf_like:
```

Comment:
I did not review the terra PR yet, but note this affects the pre-process of calls within the WDL.